vendor/github.com/Azure/azure-sdk-for-go/services/managedservices/mgmt/2019-06-01/managedservices/client.go

```diff
@@ -0,0 +1,50 @@
+// Package managedservices implements the Azure ARM Managedservices service API version 2019-06-01.
+//
+// Specification for ManagedServices.
```

Comment:
I copied the SDK files manually. Is there a way to automatically copy these files?

Reply:
Once you add the imports `go mod vendor` should do it